### PR TITLE
cpputest: init at 3.7.2

### PR DESCRIPTION
**Pull request by juliendehos**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
Hm, the version in the expression is 3.7.2?

**Comment by juliendehos** [comment]
Oops... I changed the PR title and the commit message.

File: pkgs/development/libraries/cpputest/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  version = "3.7.2";
+  name = "cpputest-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/cpputest/cpputest.github.io/blob/master/releases/${name}.tar.gz?raw=true";
```
Comment (joachifm) [inline pkgs/development/libraries/cpputest/default.nix]: Please use this url: `https://github.com/cpputest/cpputest/releases/download/${version}/${name}.tar.gz`

File: pkgs/development/libraries/cpputest/default.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    sha256 = "1xzqfl03v0nj045van62s9cpl4yliql1mfkqx1mj92c5m85y70wv";
+  };
+
+  meta = {
```
Comment (joachifm) [inline pkgs/development/libraries/cpputest/default.nix]: Please add a `license` attribute.

Comment (joachifm) [inline pkgs/development/libraries/cpputest/default.nix]: Please add yourself as the maintainer of this package.

**Comment by juliendehos** [comment]
The developers  use their own license (https://raw.githubusercontent.com/cpputest/cpputest/master/COPYING): free software, redistribution with some conditions. Should I add a new license in lib/license.nix or is there a generic entry I could use ?

**Comment by joachifm** [comment]
@juliendehos that is the 3-clause BSD license: https://opensource.org/licenses/BSD-3-Clause

**Comment by juliendehos** [comment]
Ok, thanks. License and maintainer added & url updated.

**Comment by joachifm** [comment]
Thank you

**Pull request merged**